### ISSUE TITLE
error: file '<pathToTopLevelDirectory>/<packageName>' does not exist

### ISSUE THREAD

**Issue by alexcastrounis** [pypa/packaging-problems#171]
Hello,

Thank you for the great tutorial. 

I'm getting the following error when trying to build a package. 'pathToTopLevelDirectory' below is the path where setup.py is located, and 'packageName' is the filename given, and is the same as my package name.

The package has a directory structure as follows:

root dir/
&nbsp;&nbsp;&nbsp;&nbsp;setup.py
&nbsp;&nbsp;&nbsp;&nbsp;pkg dir/
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;\__init\__.py
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;subdir1/
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;\__init\__.py
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;subdir2/
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;\__init\__.py      
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;subdir3/
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;\__init\__.py
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;subsubdir1/
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;\__init\__.py
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;subsubdir2/
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;\__init\__.py

Perhaps there is a relative vs absolute path issue? None of the __init__.py files have any import statements though, so I'm not sure why that would be if that's the case.

Any help would be greatly appreciated!

**TERMINAL OUTPUT and ERROR**

running bdist_wheel
running build
running build_py
running build_scripts
error: file '\<pathToTopLevelDirectory\>/\<packageName\>' does not exist

**Comment by merwok** [comment]
Please show where setup.py is in your tree, and what values you have for `packages` and `package_dir` in setup.py (or the whole setup.py if possible)

**Comment by acastrounis** [comment]
I've updated to show setup.py above. Here is the entry for packages, and I don't have an entry yet for ```package_dir```.

packages=setuptools.find_packages(),

**Comment by merwok** [comment]
Are you running your build command inside root dir?

**Comment by acastrounis** [comment]
Yes I am

**Comment by acastrounis** [comment]
Sorry, makes sense. Here's the entire repo with all code.

https://github.com/acastrounis/wranglyzer

**Comment by benoit-pierre** [comment]
Drop this line in `setup.py`:
```python
scripts = [''],
```

**Comment by acastrounis** [comment]
That line is in there already

**Comment by merwok** [comment]
Drop meaning remove.  The empty string is not valid here.

**Comment by acastrounis** [comment]
Fantastic, that worked. Thank you!

**Comment by benoit-pierre** [comment]
Yes, `scripts` should be a list of scripts: each entry is a path relative to your project's root directory, which is why you're getting an error about the `<pathToTopLevelDirectory>/<packageName>` file not existing.

**Comment by merwok** [comment]
This can be closed @woodruffw
